CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ Please, check out guidelines: https://keepachangelog.com/en/1.0.0/
 
 ### Fixed
 
+- Fixing build settings repeated same value https://github.com/tuist/tuist/pull/391 @platonsi
```

Comment:
oops, looks like the entry was added under the previous version, this can go to the `Next Version` section. Pulling in the latest master should include `Fixed` section from yesterday